exams/utils_test.py

```diff
@@ -123,6 +127,7 @@ def test_exam_authorization(self):
             course=self.course_run.course
         ).exists() is True
 
+    @override_settings(FEATURES={"SUPPRESS_PAYMENT_FOR_EXAM": False})
```

Comment:
we need this, if user turn this feature on using `.env` file
```shell
FEATURES_SUPPRESS_PAYMENT_FOR_EXAM=True
```

here we are making sure , this test is for `FEATURES_SUPPRESS_PAYMENT_FOR_EXAM=False`